Source/WebCore/svg/graphics/filters/SVGFilter.h

```diff
@@ -75,6 +77,7 @@ class SVGFilter final : public Filter {
     SVGUnitTypes::SVGUnitType m_primitiveUnits;
 
     SVGFilterExpression m_expression;
+    FilterEffectVector m_effects;
```

Comment:
Does the order in this vector mean anything, or is it just a way to reference the individual effects?

Reply:
The order of the effects in this vector is not really important. But the order of m_expression since it is the order of applying the filter. Each SVGFilterExpressionTerm now has an index to this vector. So when encoding the SVGFilter we have to encode both m_expression and m_effects in their orders.